docs/piscine-rust-curriculum.md

```diff
@@ -10,14 +10,14 @@ Exercices:
 
 - fibonacci2 | https://github.com/01-edu/public/tree/master/subjects/fibonacci2
 - scalar | https://github.com/01-edu/public/tree/master/subjects/scalar
-- temperature-conv | https://github.com/01-edu/public/tree/master/subjects/temperature-conv
+- temperature-conv | https://github.com/01-edu/public/tree/master/subjects/temperature_conv
```

Comment:
As this is your branch can you please do the same thing to the begging of the line for all of them?

```
- temperature_conv | https://github.com/01-edu/public/tree/master/subjects/temperature_conv
```

Reply:
oh, ok

Reply:
@MSilva95 done